sub-packages/bionemo-llm/src/bionemo/llm/run/config_models.py

```diff
@@ -46,6 +46,21 @@
 }
 
 
+def deserialize_str_to_path(path: str) -> pathlib.Path:
+    """General purpose deserialize for string/path objects. Since YAML has no native representation for pathlib.Path, we serialize to strings. Import this method as a @field_validator."""
+    return pathlib.Path(path)
+
+
+def serialize_path_or_str(path: str | pathlib.Path) -> str:
+    """General purpose serialization for string/path objects. Since YAML has no native representation for pathlib.Path, we serialize to strings. Import this method as a @field_serializer."""
+    if isinstance(path, pathlib.Path):
+        return str(path)
+    elif isinstance(path, str):
+        return path
+    else:
+        raise ValueError(f"Expected str or pathlib.Path, got {type(path)}")
+
```

Comment:
Minor suggestion: these don't necessarily need to be in `bionemo-llm`. They're good candidates to go into `bionemo-core` IMO. Not review blocking, but something to consider.